app/bundles/CoreBundle/Assets/json/regions.json

```diff
@@ -925,101 +925,107 @@
 		"Occitanie",
 		"Pays de la Loire",
 		"Provence-Alpes-Côte d'Azur",
-		"Jura",
 		"Ain",
 		"Aisne",
 		"Allier",
 		"Alpes-de-Haute-Provence",
+		"Hautes-Alpes",
 		"Alpes-Maritimes",
-		"Ard\u00e8che",
+		"Ardèche",
 		"Ardennes",
-		"Ari\u00e8ge",
+		"Ariège",
 		"Aube",
 		"Aude",
 		"Aveyron",
-		"Bas-Rhin",
-		"Bouches-du-Rh\u00f4ne",
+		"Bouches-du-Rhône",
 		"Calvados",
 		"Cantal",
 		"Charente",
 		"Charente-Maritime",
 		"Cher",
-		"Corr\u00e8ze",
+		"Corrèze",
 		"Corse-du-Sud",
-		"C\u00f4te-d'Or",
-		"C\u00f4tes-d'Armor",
+		"Haute-Corse",
+		"Côte-d'Or",
+		"Côtes-d'Armor",
 		"Creuse",
-		"Deux-S\u00e8vres",
 		"Dordogne",
 		"Doubs",
-		"Dr\u00f4me",
-		"Essonne",
+		"Drôme",
 		"Eure",
 		"Eure-et-Loir",
-		"Finist\u00e8re",
+		"Finistère",
 		"Gard",
+		"Haute-Garonne",
 		"Gers",
 		"Gironde",
-		"Haut-Rhin",
-		"Haute-Corse",
-		"Haute-Garonne",
-		"Haute-Loire",
-		"Haute-Sa\u00f4ne",
-		"Haute-Savoie",
-		"Haute-Vienne",
-		"Hautes-Alpes",
-		"Hautes-Pyr\u00e9n\u00e9es",
-		"Hauts-de-Seine",
-		"H\u00e9rault",
-		"Indre",
+		"Hérault",
 		"Ille-et-Vilaine",
+		"Indre",
 		"Indre-et-Loire",
-		"Is\u00e8re",
+		"Isère",
+		"Jura",
 		"Landes",
 		"Loir-et-Cher",
 		"Loire",
+		"Haute-Loire",
 		"Loire-Atlantique",
 		"Loiret",
 		"Lot",
 		"Lot-et-Garonne",
-		"Loz\u00e8re",
+		"Lozère",
```

Comment:
This has 1 disadvantage which is that if this value is already selected by a contact then the information will be lost with this change. Ideally, this PR should have a migration that would find and replace the old version to the new one.

Reply:
I see. In that case there is no need for a migration.